oldVersions/events/NewEvaluationListEvents.js

```diff
@@ -45,11 +45,11 @@ const NewEvaluationListEvents = function() {
 
         const instantiateObject = function(){
             
-            s.addCandidate({
-                candidate: document.getElementById("newEvaluationCandidate").value,
-                interviewer: document.getElementById("newEvaluationInterviewer").value,
-                date: document.getElementById("newElementDate").value
-            });
+            s.addCandidate([
```

Comment:
Any particular reason to use an array of values instead of a probably more intuitive object with properly named properties?